### PR TITLE
Some more doc updates

### PR DESCRIPTION
**Pull request by laf**
#### Please note

> Please read this information carefully.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

Sometimes I do wonder how other files end up in the commit, a few files here are unrelated to this and I've not committed or changed them :/

### CODE REVIEW

File: html/css/styles.css [hunk 1]
```diff
@@ -1881,6 +1881,7 @@ label {
   padding:8px;
   border-radius:5px;
   text-align:center;
+  white-space: nowrap;
```
Comment (murrant) [inline html/css/styles.css]: Intended change?

File: doc/Support/FAQ.md [hunk 2]
```diff
@@ -163,6 +164,18 @@ here are those values:
 10^15   P - peta
 ```
 
+#### <a name="faq22"> How can I move my LibreNMS install to another server?</a>
+
+If you are moving from one CPU architecture to another then you will need to dump the rrd files and re-create them. If you are in 
+this scenario then you can use [Dan Brown's migration scripts](How can I move my LibreNMS install to another server?).
```
Comment (murrant) [inline doc/Support/FAQ.md]: This link doesn't look right.

**Comment by murrant** [comment]
This usually happens to me when I mess up my master :/

**Comment by laf** [comment]
I'd agree

**Pull request closed without merging**